python/tvm/relax/frontend/torch/base_fx_graph_translator.py

```diff
@@ -1416,6 +1416,19 @@ def _empty_like(self, node: fx.Node) -> relax.Var:
         x = self.env[node.args[0]]
         return self.block_builder.emit(relax.op.zeros_like(x))
 
+    def _eye(self, node: fx.Node) -> relax.Var:
+        args = self.retrieve_args(node)
+        if len(args) == 1:
+            n = args[0]
+            m = n
+        elif len(args) == 2:
+            n = args[0]
+            m = args[1]
+        else:
```

Comment:
I don't think we need to take care of this condition because the pytorch exporter does that.
So we can assume that the graph in the exported program doesn't contain wrong nodes.
Probably this is enough?
```python
n = args[0]
m = args[1] if len(args) > 1 else n
```

cc @Hzfengsy